onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6758,40 +6758,56 @@ def aten_view(self: TTensor, size: IntType) -> TTensor:
     return op.Reshape(self, size)
 
 
+@torch_op("aten::view_as")
 def aten_view_as(self: TensorType, other: TensorType) -> TensorType:
     """view_as(Tensor(a) self, Tensor other) -> Tensor(a)"""
 
-    raise NotImplementedError()
+    size = op.Shape(other)
+    return op.Reshape(self, size)
 
 
+@torch_op("aten::view_as_complex")
 def aten_view_as_complex(self: TensorType) -> TensorType:
```

Comment:
Why not update TensorType to something in tensor_typing.py? I thought TensorType should always be changed?

Reply:
Good point. Thanks!

Reply:
Done. Updated to TTensor